use-cases/client-server-msgs-perf/svmsg_file.h

```diff
@@ -38,7 +38,8 @@
 /**
  * Message Types for request / response messages sent from server to client
  */
-typedef enum {
+typedef long req_resp_type_t;
```

Comment:
Cleaning up an existing inconsistency.

As per the [msgsnd() man](https://www.man7.org/linux/man-pages/man3/msgsnd.3p.html) page, we need the message structure to be defined as:

```
       The application shall ensure that the argument msgp points to a
       user-defined buffer that contains first a field of type long
       specifying the type of the message, and then a data portion that
       holds the data bytes of the message. The structure below is an
       example of what this user-defined buffer might look like:

           struct mymsg {
               long   mtype;       /* Message type. */
               char   mtext[1];    /* Message text. */
           }
```

I have no idea how things used to work before, but this seems to be the right thing to do.

Kindly do cross-check and verify that this change is kosher.